### ISSUE TITLE
Render Z1 - Z4 more interesting

### ISSUE THREAD

**Issue by GeeKaa71** [gravitystorm/openstreetmap-carto#4740]
I don't know if this is the right place for this proposal, but my thought is the following.  
At the moment the lower zoom levels (1 to 4) are a bit boring. I propose to color the continents on these levels according to the example below. I assume 7 continents (Asia, Africa, North America, South America, Antarctica, Europe, Oceania (including Australia) that we can give a separate color.  
Zooming in further (up to and including zoom level 4) then gives a different shade per country. Optionally, we can have zoom levels 5 and 6 fade out the colors (for example, by adjusting the saturation for these levels).  

[wikipedia Lijst_van_landen_per_werelddeel](https://nl.wikipedia.org/wiki/Lijst_van_landen_per_werelddeel)

The advantages of this coloring:  
The zoom levels 1 to 4 become more exciting and fun to discover.  
It helps to recognize the different countries and continents and develops our geographical knowledge.  
![ex-det1](https://user-images.githubusercontent.com/119065086/204815779-e9234013-f60b-440d-a87f-787fc9776b42.png)

**Comment by imagico** [comment]
First of all:  Yes, this is the right place for suggestions like that.

Some general comments explaining the context:

* our past considerations and discussions for the low zoom levels are nicely summarized by @jeisenbe on: https://github.com/gravitystorm/openstreetmap-carto/issues/2688#issuecomment-597929915.
* OSM AFAIK has no data on continental divisions of the planet because such data is clearly [non-verifiable](https://wiki.openstreetmap.org/wiki/Verifiability).
* At zoom levels 1-3 we currently do not use OSM data for the shown administrative boundaries - see #2172.  We would need to change that (preferably in a way that would also work at z4 to about z7 and for admin_level 3/4) before we can think about doing anything more fancy with the administrative boundaries at those zoom levels.
* being exciting and fun to discover is not among our [goals](https://github.com/gravitystorm/openstreetmap-carto/blob/master/CARTOGRAPHY.md).

Personally i think that the idea of displaying a political map, even if it is only at the lowest zoom levels, would not work well towards our [goals](https://github.com/gravitystorm/openstreetmap-carto/blob/master/CARTOGRAPHY.md).  My suggestion was and is to reduce the focus on administrative boundaries and increase the focus on rendering the physical geography at the low zoom levels because that is more important for mapper feedback.  It also would better highlight and get users interested in what OSM is primarily about (that is mapping of the locally verifiable geography).  The administrative boundary data is - even if we'd actually use it at these scales - nothing where OSM has anything like a unique selling point that can be in any way shown or even hinted at at these scales.

Quite a few years back meanwhile i presented a concept how a low zoom level extension of an OSM-Carto like map style could look like:

https://imagico.de/maps/#map=4/49.781/12.568&lang=en&r=osmlz&o=aa&ui=8

**Comment by GeeKaa71** [comment]
Hi,
Thanks for the comment. I think your imagico.de concept already looks a lot better than the map on the homepage of openstreetmap.org.

Fun may not be an end in itself, but it does help to attract new users. I wouldn't be here without having fun in this project :-).

I think it is certain that the zoom levels 1-4 could use some attention. I also find an important advantage that colouring in different countries helps to find countries. Now many people do not know where (for example) Sierra Leone is located. Colouring the map makes it (I think) easier to find individual countries and places.

Anyway, they were just ideas, worth considering I think.

**Comment by imagico** [comment]
Don't get me wrong, it would definitely be important to have more fun oriented map layers on osm.org.  But it is hard as it is to consolidate the different, often conflicting goals we have in a uniform design and pursuing the fun factor as an additional goal in addition in the same map style that also pursues many other, more serious goals is IMO not such a good idea.  The fun aspect would be better served with a dedicated map style.

The most important criteria that i see for improving the low zoom levels specifically (in addition the the goals we have for the map at all zoom levels) are:

* providing the user broad orientation about the geography.
* pointing out the broad structure in mapping coverage (in particular where there are gaps and where there are inconsistencies in mapping).
* potentially showing large size errors that would otherwise go unnoticed because they are in areas most users don't zoom into further.

As is the CyclOSM map style seems to serve this goal much better than OSM-Carto:

https://www.openstreetmap.org/#map=3/0.00/0.00&layers=Y

so we definitely have work to do here.  This is challenging though because in most aspects this requires preprocessing of data, which is a lot of work to develop and maintain with regular updates.  Any help with that would certainly be welcome.

**Comment by GeeKaa71** [comment]
Hi, i agree on this, so lets close this as a wontfix... :-) just one question about this, what kind of pre-processing of data is required? Is this something humans have to do, or is this some automated task? Greetz...
